Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/configs/LinkConfig.java

```diff
@@ -16,6 +16,8 @@
 public class LinkConfig {
     private String reportabilityPredicate;
 
+    private boolean cqlDebug = false;
```

Comment:
Is there still a use case for this to be set to true?  Give the volume of logging that this produces and the performance impact of trawling the CQL for each log message (or maybe that's not an issue?), do we want to restrict debugging to either the CLI or explicit calls to `POST /api/measure-definition/{id}/$evaluate`?

Reply:
I think that's a good consideration in the future... But, I'd like to keep `cqlDebug` configuration _for now_ until we have a way to get the exact patient bundle that's used for evaluation out of the system that _could_ be used with `/$evaluate`... As far as I know, there's no way to ask the system to store the patient bundles that are input to the measure eval, so that we could grab those and test against them with `/$evaluate`.